### PR TITLE
Adding a guide that is run the first time a user enters catwalk.

### LINKED ISSUE #16: Develop a UI walkthrough in the tool
Similar to https://github.com/qlik-oss/core-experimenter

### PR DESCRIPTION
**Pull request by hrigner**
Closes #16

### CODE REVIEW

**Comment by sublibra** [comment]
Looks very good overall!

Step 2:
Need a stronger wrap on the URL?
image

Step 3:
Picky picky.. rounded corner and arrow so close seems like a visual bug to me.
image

If you open an app without data the flow is broken and you are thrown to hypercubebuilder step after step 2. The guide breaks after this.

**Comment by sublibra** [comment]
Sorry SBS

**Comment by peol** [comment]
@sublibra Usually people use `<input>` elements to wrap long links that should be easy to copy/paste.

File: src/components/guide-steps.jsx [hunk 1]
```diff
@@ -0,0 +1,237 @@
...
+    content: (
+      <div>
+        <p>
+          Fields, the data-carrying entities in the data model, are represented with a box like this.
+        </p>
+        <p>
+          The field name and the number of field values are visible.
+        </p>
+        <p>
+          The number of field values are presented in the form of 6 of 6(5)
```
Comment (peol) [inline src/components/guide-steps.jsx]: This section reads a bit weird for me, shouldn't the full form be presented first before talking about parts of it? I.e. explain `6(5)` before explaining `6`. I also got a bit confused when the numbers weren't related to my app/the focused field :)

**Pull request merged**